lib/cocoapods-fix-react-native/versions/0_54_4.rb

```diff
@@ -76,8 +76,41 @@ def fix_unused_yoga_headers
   end
 end
 
+# Detect source file dependency in the generated Pods.xcodeproj workspace sub-project
+def has_pods_project_source_file(source_filename)
+  pods_project = 'Pods/Pods.xcodeproj/project.pbxproj'
+  if File.foreach(pods_project).grep(/#{source_filename}/).any?
+    return true
+  end
+  return false
+end
```

Comment:
This can be replaced with a 2-liner:

```ruby
def has_pods_project_source_file(source_filename)
  pods_project = 'Pods/Pods.xcodeproj/project.pbxproj'
  File.read(pods_project).grep(/#{source_filename}/)
end
```

As it's only one file, so no need for the array-ness 👍

Reply:
<face meet palm> Totally !  Nice cleanup thank you Orta.